### PR TITLE
💬 Add style for pull-quotes

### PR DESCRIPTION
**Pull request by agoose77**
I'm not sure if this is best-practice; I'm implying that a pull-quote really is a container-level feature rather than a blockquote-level feature. That follows from how they're implemented in https://github.com/executablebooks/mystmd/pull/961

### CODE REVIEW

**Review by rowanc1 — CHANGES_REQUESTED** [review]
Simple so far! This should be put in the `figures.css` file though I think, as these are additional styles, not overidding tailwind (that is also where other figure css is defined).

**Comment by rowanc1** [comment]
We should revert part of this PR to get the kind back on the figure node. maybe prefixing it with `fig-${kind}` or something?

https://github.com/executablebooks/myst-theme/pull/277

And then changing the CSS here:
https://github.com/executablebooks/myst-theme/blob/main/styles/figures.css#L1

Both the `code` and `quote` container kinds are a bit degraded, we can't use the class `table` though, so some sort of prefix would be good.

**Comment by agoose77** [comment]
So, I'm guessing that #277 was a deliberate "regression" due to the bare `kind` class being misinterpreted? And, thus, you're suggesting we introduce a custom name like `figure-${kind}` to disambiguate this.

**Review by rowanc1 — CHANGES_REQUESTED** [review]
A few more things on review!

File: packages/myst-to-react/src/basic.tsx
Comment (agoose77) [reply packages/myst-to-react/src/basic.tsx:221]: @rowanc1 thanks for the hand-holding on this. I tend to work in a "push first", "check later" style before merging, so you'll probably see a few things like this.

Comment (rowanc1) [reply packages/myst-to-react/src/basic.tsx:221]: Realize that this was a few minutes after you pushed - I am having coffee, waking up and doing pr reviews on my side... ☕️

**Review by rowanc1 — APPROVED** [review]
Ok - just changesets and this is good from my perspective - can you show a picture of the pullquote and maybe @choldgraf can comment on styles. :)

**Comment by agoose77** [comment]
We've added two special kinds of quotes in executablebooks/mystmd#961:

Epigraph:
image

Pull-quote:
image

Currently, only pull-quotes are differently styled here, with a larger typeface. We could do more, such as background "quote" characters, but I don't have strong opinions.

**Comment by choldgraf** [comment]
It's a bit hard to judge design without seeing a live site with pull quotes used in a "real world" scenario, but these seem like a reasonable start. Personally I'd make the pull quote a little bit bigger.

As an example, [here are several examples and inspiration for pull quotes](https://creativepro.com/how-to-attract-attention-pull-quotes/).

Can pull quotes be **placed** a-la columns etc? I often see pull quotes as a sidebar with the text floating around it. For example, here's one from the NYT:

image

I don't think that needs to block this PR but just noting it as a common use-case.

File: styles/figures.css
Comment (rowanc1) [inline styles/figures.css:9]: In addition to being a bit bigger, I think that these should also be in rems!

```suggestion
  font-size: 1.25rem;
```

**Comment by rowanc1** [comment]
@choldgraf I have moved this into an issue to track your sources: #329.

Let's increase the size and get it in (after the other PR merges!). We have added the ability to also add classes, which eventually should be able to style these in the near future....!

**Pull request merged**
